app/controllers/report_controller.rb

```diff
@@ -178,6 +178,10 @@ def x_show
   def tree_select
     @edit = nil
     @sb[:select_node] = false
+    if @sb.try(:[], :flash_msg)
```

Comment:
btw. not sure what version of ruby, we support right now, but the best is using safe navigation operator .& ... 

>  I haven't seen this @sb.try(:[], ...) construct anywhere
in our code, I consider it a bad coding practice and I won't merge this PR looking like this.

```$ grep -rn \\.try app/ | wc -l``` 458  ;-)